apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/BooleanExpression.kt

```diff
@@ -96,6 +98,7 @@ fun <T : Any> or(vararg other: BooleanExpression<T>): BooleanExpression<T> = Boo
 fun <T : Any> and(vararg other: BooleanExpression<T>): BooleanExpression<T> = BooleanExpression.And((other.toList()).toSet())
 fun <T : Any> not(other: BooleanExpression<T>): BooleanExpression<T> = BooleanExpression.Not(other)
 fun variable(name: String): BooleanExpression<BVariable> = BooleanExpression.Element(BVariable(name))
+fun label(label: String?): BooleanExpression<BLabel> = BooleanExpression.Element(BLabel(label))
```

Comment:
You could make it `label(label: String? = null)` so that the generated code reads nicer when there is no label:

```
label().evaluate(...)
```